node_modules/oae-util/tests/test-server.js

```diff
@@ -155,4 +156,43 @@ describe('OAE Server', function() {
             });
         });
     });
-});
\ No newline at end of file
+
+    describe('Cookies', function() {
+
+        /**
+         * Verify that a fixed cookie name is used
+         */
+        it('verify a fixed cookie name is used', function(callback) {
+            TestsUtil.generateTestUsers(camAdminRestContext, 1, function(err, user, simong) {
+                assert.ok(!err);
+
+                // Ensure the user is authenticated
+                PrincipalsTestUtil.assertGetMeSucceeds(simong.restContext, function(me) {
+                    assert.ok(!me.anon)
+
+                    // Get the cookies from the cookie har
+                    var cookies = simong.restContext.cookieJar.getCookies('http://localhost');
+                    var cookieNames = _.pluck(cookies, 'key');
+
+                    // Verify the configured cookie names are being used
+                    var config = require('../../../config.js').config;
+                    assert.ok(_.contains(cookieNames, config.cookie.name));
+
+                    // Rename the cookie to something else
+                    _.each(cookies, function(cookie) {
+                        if (cookie.key === config.cookie.name) {
+                            cookie.key = 'somethingElse';
+                        }
+                    });
```

Comment:
Rather than changing the cookie on the client, I was referring to changing the `config.cookie.name` value in the initial test setup to something like `somethingElse`. I want to avoid the case where express has another major version change in which our `config.cookie.name` is no longer taking effect and our tests don't catch it because we're using the default.